commcare-cloud/commcare_cloud/commands/ansible/ansible_playbook.py

```diff
@@ -229,3 +235,198 @@ def run(self, args, unknown_args):
+            args.module_args = "name=%s state=%s" % (
+                self.ANSIBLE_MODULE_FOR_SERVICE.get(service, service),
+                state)
+            RunAnsibleModule(self.parser).run(
+                args,
+                unknown_args
+            )
+
+    def run_for_services(self, service_group, args):
+        if args.only:
+            return args.only.split(',')
+        else:
+            return self.SERVICES[service_group]
+
+    def run_for_es(self, args, unknown_args):
+        action = args.action
+        if action == "restart":
+            RestartElasticsearch(self.parser).run(args, unknown_args)
+        else:
+            self.run_ansible_module_for_service_group("es", args, unknown_args)
+
+    def run_for_riakcs(self, args, unknown_args):
+        tags = []
+        action = args.action
+        state = self.DESIRED_STATE_FOR_ACTION[action]
+        args.playbook = "service_playbooks/riakcs.yml"
+        run_for_services = self.run_for_services('riakcs', args)
+        if args.only:
+            # for options to act on certain services create tags
+            for service in run_for_services:
+                if service:
+                    tags.append("%s_%s" % (action, service))
+            if tags:
+                unknown_args.append('--tags=%s' % ','.join(tags),)
+        unknown_args.extend(['--extra-vars', "desired_state=%s desired_action=%s" % (state, action)])
+        # ToDo: use this with when in the playbook instead of tags
+        # currently its running riak even when just riak-cs is ran
+        # but skips riak when running for only stanchion
+        # as of now it looks like
+        # --extra-vars 'desired_services=['"'"'riak-cs'"'"', '"'"'stanchion'"'"']'
+        unknown_args.extend(['--extra-vars', "desired_services=%s" % run_for_services])
+        AnsiblePlaybook(self.parser).run(args, unknown_args)
+
+    def run_for_kafka(self, args, unknown_args):
+        tags = []
+        action = args.action
+        state = self.DESIRED_STATE_FOR_ACTION[action]
+        args.playbook = "service_playbooks/kafka.yml"
+        run_for_services = self.run_for_services('kafka', args)
+        if args.only:
+            # for options to act on certain services create tags
+            for service in run_for_services:
+                if service:
+                    tags.append("%s_%s" % (action, service))
+            if tags:
+                unknown_args.append('--tags=%s' % ','.join(tags),)
+        unknown_args.extend(['--extra-vars', "desired_state=%s desired_action=%s" % (state, action)])
+        # ToDo: use this with when in the playbook instead of tags
+        unknown_args.extend(['--extra-vars', "desired_services=%s" % run_for_services])
+        AnsiblePlaybook(self.parser).run(args, unknown_args)
+
+    def run_for_pg_standby(self, args, unknown_args):
+        self.run_ansible_module_for_service_group('pg_standby', args, unknown_args, inventory_group="pg_standby")
```

Comment:
This is the only place we pass in the `inventory_group` arg? If so, I don't see what it gets us, since I think if we left it `None`, the default behavior would arrive at the same answer. If I'm not missing something, I'd suggest removing this argument from the method.

Reply:
@dannyroberts 
Since `pg_standby` also checks for `postgresql` it would end up checking for inventory group for service `postgresql` when calling `get_inventory_group_for_service`. As of now it probably would work without this `inventory_group ` argument since `get_inventory_group_for_service` will fall back to service group which is `pg_standby`, but i still chose to do this to make it an explicit override from the normal behaviour.
Fine to remove if you think its not needed or wont be needed.

Reply:
OK. I'm not sure I totally understand, but if you have a good reason in mind for keeping it, I'm fine with that. If we end up not needing it, it's easy to remove later as well.